### ISSUE TITLE
Having the "Calculate Shapely Surfaces" option saved

### ISSUE THREAD

**Issue by chopinregis** [IfcOpenShell/IfcOpenShell#3530]
Currently when ever this option is checked, when we save the ifc project/file, this doesn't get saved and we constantly have to go back to check the box. It would be nice if this option is also saved when ever we save the file. 

![Screenshot 2023-07-31 170950](https://github.com/IfcOpenShell/IfcOpenShell/assets/13260872/07a5b4d9-1110-4895-8cd7-0cef30d3ae0c)

**Comment by Moult** [comment]
Yes, this is due to surface calculation still being an experimental feature. I guess we should upgrade it :)

**Comment by sboddy** [comment]
UI has long since changed. Now a Fill Mode dropdown to select whether/which fill method to use. This selection is remembered. Closing.
